### ISSUE TITLE
Override Windows or other programs hotkeys while in vscode

### ISSUE THREAD

**Issue by zenVentzi** [Microsoft/vscode#57095]
I don't know if that's even slightly possible.Problem is some of VSCode's hotkeys are not working. Although many of them seem to do nothing at first glance, they are most likely assigned to some other program/process or Windows' feature. Is it possible while in VSCode when using those hotkeys(e.g. alt+shift+i is not working and I don't know what it is being used for and by what program/process) to use the VSCode one, without changing it to some other?

In other words, when I minimize VSCode, these hotkeys to continue working as intended, but when in VSCode, to use the VSCode comamnds.

**Comment by bpasero** [comment]
This is currently not possible and since I am not sure how to even implement a solution here, I am closing this.
